### DISCUSSION TITLE
Unwanted Status Line Color Change with Telescope

### DISCUSSION THREAD

**Discussion by Doodler8888** [rose-pine/neovim#169] [General]
I've used the following settings to disable background colors:

```
disable_background = true,
disable_float_background = true,
```
However, when I open a Telescope window, the color of the Neovim status line changes to match the disabled background color. Is there a way to disable or modify this behavior?

**Comment by mvllow** [comment]
This happens even without `disable_[float]_background = true`. Try linking `StatusLineNC` to `StatusLine`:

```lua
require("rose-pine").setup({
	highlight_groups = {
		StatusLineNC = { link = "StatusLine" },
	},
})
```

**Reply by Doodler8888** [reply]
Works perfect, thanks!
